src/V3Assert.cpp

```diff
@@ -20,23 +20,15 @@
 
 #include "V3Stats.h"
 
-#include <limits>
 VL_DEFINE_DEBUG_FUNCTIONS;
 
 //######################################################################
 // Assert class functions
 
 class AssertVisitor final : public VNVisitor {
     // TYPES
-    enum class DirectiveType : uint8_t {
-        INTRINSIC,  // AstNodeAssertIntrinsinc
-        CASE,  // unique/unique0/priority case related checks
-        IF,  // unique/unique0/priority if related checks
-        ASSERT,  // assert statement
-        COVER,  // cover statement
-        ASSUME,  // assume statement
-    };
     using VAssertType_t = std::underlying_type<VAssertType::en>::type;
+    using VAssertDirectiveType_t = std::underlying_type<VAssertDirectiveType::en>::type;
```

Comment:
Why can't we use VAssertDirectiveType directly (as we do everywhere else)?  Probably using VAssertType_t is likewise not needed.  Likewise this file shouldn't need to be using VAssertDirectiveType::en.